Trappist/src/Promact.Trappist.DomainModel/Models/Report/Report.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Promact.Trappist.DomainModel.Enum;
+using Promact.Trappist.DomainModel.Models.TestConduct;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Promact.Trappist.DomainModel.Models.Report
+{
+    public class Report : BaseModel
+    {
+        [Required]
+        public double TotalMarksScored { get; set; }
```

Comment:
reason for making it require? double/int/float datatypes are not nullable by default. it takes default values if not provided explicitly. So wouldn't make difference giving required attribute.